CHANGELOG.md

```diff
@@ -388,7 +388,7 @@ This is the first stable release of Lingua Franca. Aside from numerous bugfixes,
 - Multiports and banks are now iterable in the Python target ([#713](https://github.com/lf-lang/lingua-franca/pull/713))
 - Fixed an issue where top-level custom Python classes were being serialized incorrectly
 - `bank_index` (useful for banks of reactors) is now a proper parameter ([#424](https://github.com/lf-lang/lingua-franca/pull/424))
-  that can be passed down the reactor hierarchy via parameter assignment.
+that can be passed down the reactor hierarchy via parameter assignment.
```

Comment:
This change isn't right... Looks like it's part of the sentence that starts with the previous list item.

Reply:
This change was made automatically. I think that it is semantically equivalent Markdown and that the removal of this indentation should not separate it from the list. ([Here is how it looks on GitHub.](https://github.com/lf-lang/lingua-franca/blob/update-lf-code-width/CHANGELOG.md)) To separate it from the list, I think you would have to add a newline.

Reply:
That may be, but that just means it wasn't right in the first place :-) Let's correct it...

Reply:
I'm confused because it looks right to me as rendered on GitHub. This last line appears as part of the list item that starts with `bank_index`.

Reply:
To be clear, something was off about that list item, but it wasn't due to the formatting changes in this PR.

Reply:
Marking this as resolved by the latest commit on this branch.